lib/src/html/template_data.dart

```diff
@@ -56,12 +56,7 @@ abstract class TemplateData<T extends Documentable> {
 
   Iterable<Subnav> getSubNavItems() => <Subnav>[];
 
-  String _layoutTitle(String name, String kind, bool isDeprecated) {
-    if (kind.isEmpty) kind = '&nbsp;';
-    String str = '<span class="kind">$kind</span>';
-    if (!isDeprecated) return '${str} ${name}';
-    return '${str} <span class="deprecated">$name</span>';
-  }
+  String _layoutTitle(String name, String kind) => '${kind} ${name}';
```

Comment:
updated the PR; here's the latest:

<img width="343" alt="screen shot 2017-05-15 at 7 50 49 pm" src="https://cloud.githubusercontent.com/assets/1269969/26087853/57479fec-39a8-11e7-9d6c-a6bc3d1f8230.png">